src/modules/0.0.21/iasql_functions/rpcs/iasql_apply.ts

```diff
@@ -0,0 +1,36 @@
+import { snakeCase } from 'typeorm/util/StringUtils';
+
+import { IasqlFunctions } from '..';
+import { Context, RpcBase, RpcResponseObject } from '../../../interfaces';
+import * as iasql from '../iasql';
+
+export class IasqlApply extends RpcBase {
+  module: IasqlFunctions;
+  name = 'iasql_apply';
+  output = {
+    action: 'varchar',
+    table_name: 'varchar',
+    id: 'varchar',
+    description: 'varchar',
+  } as const;
+  call = async (
+    dbId: string,
+    _dbUser: string,
+    _ctx: Context,
+  ): Promise<RpcResponseObject<typeof this.output>[]> => {
+    const applyRes = (await iasql.apply(dbId, false)).rows;
+    const formattedRes = applyRes.map(rec =>
+      Object.keys(rec).reduce((acc, key) => {
+        acc[snakeCase(key)] = rec[key];
+        return acc;
+      }, {} as any),
```

Comment:
Not a big nit, but in case you weren't aware, you can also write this as:

```suggestion
      Object.fromEntries(Object.entries(rec).map((k, v) => [snakeCase(k), v]))
```

Reply:
oh it was the first thing that came to my mind, yours seems cleaner, I can test it